swap_meet/item.py

```diff
@@ -1,2 +1,30 @@
 class Item:
-    pass
+    def __init__(self, category = "", condition = 0.0,):
+        self.category = category
+        self.condition = condition
+
+    # Doing this made all the tests fail due to missing arguments, but
+    # To add the age attribute, I think that I would:
+    # def __init__(self, age, category = "", condition = 0.0,):
```

Comment:
Since you're grafting an additional parameter on at the end, if you don't want to disturb all the places where the existing constructor is called with positional parameters, you'd generally want to add the new parameter at the end of the existing parameters. You'd also want to give it a default value, otherwise python is going to complain about a mismatch in the number of parameters from any of the existing calls. So you'd end up with something like:

```python
    def __init__(self, category = "", condition = 0.0, age=0):
```